catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -80,7 +80,15 @@ class MDCNodeListViewController: CBCNodeListViewController {
     // swiftlint:enable force_cast
```

Comment:
Should this be removed if we are removing '!'?

Reply:
I think it should stay, we disable it for the for cast on orderedNodes then reapply it. If we keep it disabled then later changes would allow for force_cast lower in the file. This makes sure we don't add any.